spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/ConfigMapPropertySourceLocator.java

```diff
@@ -80,6 +84,12 @@ protected abstract MapPropertySource getMapPropertySource(String applicationName
 		return null;
 	}
 
+	@Override
+	@Retryable(interceptor = "configMapPropertiesRetryInterceptor")
```

Comment:
you are overriding this only for the sake of `@Retryable`. You are doing this as a safety net, correct? So that just in case we ever use `locateCollection`, it will have that `@Retryable`? If so, this is a great idea (I would have missed it if it was me doing this PR!).

Reply:
Thank you and yes. I'm overriding `locateCollection` to mark it with `@Retryable`.